linalg/vector.hpp

```diff
@@ -553,8 +558,14 @@ inline void Vector::NewMemoryAndSize(const Memory<double> &mem, int s,
 {
    data.Delete();
    size = s;
-   data = mem;
-   if (!own_mem) { data.ClearOwnerFlags(); }
+   if (own_mem)
+   {
+      data = mem;
```

Comment:
I prefer not to add `mem.ClearOwnerFlags();` -- it is unnecessary and from the call signature it is clear that ownership is given to the `Vector` because the parameter `own_mem == true` is explicitly given.

Reply:
It is clear to who? Not to MFEM, since there is two owners after this call, and not to anyone that hasn't read your note... The ownership is explicitly given "in theory", in practice it is not. Why leaving the door open for a bug when we could write clean code?